0003-Disable-statx-for-all-builds.-JB-50106.patch

```diff
@@ -1,7 +1,7 @@
-From f131514903bb02f5f9f1d312aea19e954ab1ffee Mon Sep 17 00:00:00 2001
-From: =?UTF-8?q?Tomi=20Lepp=C3=A4nen?= <tomi.leppanen@jolla.com>
-Date: Fri, 26 Jun 2020 11:58:19 +0300
-Subject: [PATCH 3/4] Disable statx for all builds. JB#50106
+From 0ea195274d580ada6af1f9687647564d9a06ecd5 Mon Sep 17 00:00:00 2001
```

Comment:
No need to update necessarily now but more like a tip for the future: use --zero-commit as well when formatting patches "git format-patch --zero-commit -N" 

Add it earlier here: https://sailfishos.org/wiki/Usage_of_packaging_formats.